docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.8"
+services:
+  saturn-node:
+    image: ghcr.io/filecoin-saturn/l1-node:${SATURN_NETWORK:-test}
+    container_name: saturn-node
+    restart: always
+    environment:
+      SATURN_NETWORK: "${SATURN_NETWORK:?please make sure to set your SATURN_NETWORK environment variable with either 'test' or 'main' in the .env file}"
+      FIL_WALLET_ADDRESS: "${FIL_WALLET_ADDRESS:?please make sure to set your FIL_WALLET_ADDRESS environment variable in the .env file}"
+      NODE_OPERATOR_EMAIL: "${NODE_OPERATOR_EMAIL:?please make sure to set your NODE_OPERATOR_EMAIL environment variable in the .env file}"
+      SPEEDTEST_SERVER_CONFIG: "${SPEEDTEST_SERVER_CONFIG}"
+    ulimits:
+        nofile:
+          soft: 65536
+          hard: 65536
+    ports:
+      - 80:80
+      - 443:443
+    network_mode: host
+    stop_signal: SIGTERM
```

Comment:
Does this override the Dockerfile one? (`SIGQUIT`)?

This (`SIGTERM`) is our drain signal

Reply:
Yes it does